openmdao/docs/basic_guide/basic_case_recording.rst

```diff
@@ -0,0 +1,26 @@
+.. _basic_case_recording:
+
+************************
+Basic Recording Example
+************************
+
+Recording Terminology
+---------------------
+
+| **Case**: A Case stores a snapshopt of all the variable values, metadata, and options of a model, or a sub-set of a model, at a particular point in time
+| **Case Recorder**: An OpenMDAO module used to store the state of a `Problem` before, during, or after execution in an SQL file.
```

Comment:
All the state is in the model.. so "state of a model" is probably more correct, 
but did we want to not use the word "state" here as well?